pkg/strategy/grid2/metrics.go

```diff
@@ -94,6 +125,62 @@ func initMetrics(extendedLabels []string) {
 			"symbol",   // symbol of the market
 		}, extendedLabels...),
 	)
+
+	metricsGridUpperPrice = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "bbgo_grid2_grid_upper_price",
+			Help: "the upper price of grid",
+		},
+		append([]string{
+			"exchange", // exchange name
+			"symbol",   // symbol of the market
+		}, extendedLabels...),
+	)
+
+	metricsGridLowerPrice = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "bbgo_grid2_grid_lower_price",
+			Help: "the lower price of grid",
+		},
+		append([]string{
+			"exchange", // exchange name
+			"symbol",   // symbol of the market
+		}, extendedLabels...),
+	)
+
+	metricsGridQuoteInvestment = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "bbgo_grid2_grid_quote_investment",
+			Help: "the quote investment of grid",
+		},
+		append([]string{
+			"exchange", // exchange name
+			"symbol",   // symbol of the market
+		}, extendedLabels...),
+	)
+
+	metricsGridBaseInvestment = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "bbgo_grid2_grid_base_investment",
+			Help: "the base investment of grid",
+		},
+		append([]string{
+			"exchange", // exchange name
+			"symbol",   // symbol of the market
+		}, extendedLabels...),
+	)
+
+	metricsGridFilledOrderPrice = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "bbgo_grid2_grid_filled_order_price",
```

Comment:
maybe you can remove the second grid keyword? it's just a minor suggestion

Reply:
removed.